### PR TITLE
docs: schedule newsletter and email group enhancement

### PR DESCRIPTION
**Pull request by Mangesh-Khairnar**
- Email group
email group
- Newsletter
newsletter-docs
docs for feat: https://github.com/frappe/frappe/pull/9603

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/CRM/newsletter.md [hunk 1]
```diff
@@ -12,6 +12,61 @@ You need to create [Email Groups](/docs/user/manual/en/CRM/email_group) before s
 To access the Newsletter list, go to:
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/CRM/newsletter.md:12]: @Mangesh-Khairnar, this file's contents are duplicated, update according to what you want to keep

Comment (Mangesh-Khairnar) [reply erpnext_documentation/www/docs/user/manual/en/CRM/newsletter.md]: added changes

File: erpnext_documentation/www/docs/user/manual/en/CRM/email_group.md [hunk 2]
```diff
@@ -28,4 +28,13 @@ You can import email IDs from existing transactions.
 
 Click on Action > Import Subscribers and select a transaction like Lead. Click on Import and all the email IDs mentioned in the email IDs will be imported.
 
-Next: [Automate Lead Capturing](/docs/user/manual/en/CRM/articles/automate_lead_capturing)
+### 2.2 Email Templates
+
+You can specify the confirmation email and welcome email template for the email group. A confirmation email will be sent to confirm a person's subscription to your newsletter. The welcome email is sent when the person confirms their subscription to your email newsletter.
+
+**a. Confirmation Email Template:** This email template is used to replace the default template sent for newsletter subscription.<br />
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/CRM/email_group.md:35]:
```suggestion
**Confirmation Email Template:** This email template is used to replace the default template sent for newsletter subscription.
```

File: erpnext_documentation/www/docs/user/manual/en/CRM/email_group.md [hunk 3]
```diff
@@ -28,4 +28,13 @@ You can import email IDs from existing transactions.
...
+**b. Welcome Email Template:** This email template defines the email that is sent whenever a subscriber is added to the email group.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/CRM/email_group.md:36]:
```suggestion
**Welcome Email Template:** Whenever a subscriber is added to an email group, the template set here will be sent to them.
```

File: erpnext_documentation/www/docs/user/manual/en/CRM/email_group.md (hunk 2 above)
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/CRM/email_group.md:35]: is this only for website or manual add? auto add via lead? add that information

File: erpnext_documentation/www/docs/user/manual/en/CRM/email_group.md [hunk 4]
```diff
@@ -28,4 +28,13 @@ You can import email IDs from existing transactions.
 
 Click on Action > Import Subscribers and select a transaction like Lead. Click on Import and all the email IDs mentioned in the email IDs will be imported.
 
-Next: [Automate Lead Capturing](/docs/user/manual/en/CRM/articles/automate_lead_capturing)
+### 2.2 Email Templates
+
+You can specify the confirmation email and welcome email template for the email group. A confirmation email will be sent to confirm a person's subscription to your newsletter. The welcome email is sent when the person confirms their subscription to your email newsletter.
+
+**Confirmation Email Template:** This email template is used to replace the default template sent for newsletter subscription. By default, email generated via this template is sent on subscription via email group Website.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/CRM/email_group.md:35]:
```suggestion
**Confirmation Email Template:** This email template is used to replace the default template sent for newsletter subscription. By default, a confirmation email generated via this template is sent when someone subscribes to the Email Group Website.
```

**Pull request merged**